package.json

```diff
@@ -12,12 +12,13 @@
     "type": "git",
     "url": "git+https://github.com/govau/datavizkit.git"
   },
-  "main": "index.js",
+  "main": "lib/Datavizkit.js",
```

Comment:
@micapam if you change this locally to "src/Datavizkit.js"` you'll be able to work locally when in dto-dashboard scope